subgraphs/spotify/src/utils/constants.ts

```diff
@@ -0,0 +1,23 @@
+export const OAUTH_SCOPES = [
+  'streaming',
+  'user-follow-modify',
+  'user-follow-read',
+  'playlist-read-private',
+  'playlist-modify-private',
+  'playlist-read-collaborative',
+  'playlist-modify-public',
+  'user-modify-playback-state',
+  'user-read-email',
+  'user-read-currently-playing',
+  'user-read-playback-position',
+  'user-read-playback-state',
+  'user-library-read',
+  'user-library-modify',
+  'user-top-read',
+  'user-read-recently-played',
+] as const;
+
+export const TOPICS = {
```

Comment:
Are these needed in this subgraph since these are used for the subscription?

Reply:
Nope, deleted